test/spec/subaccounts.speq.js

```diff
@@ -0,0 +1,165 @@
+var chai = require('chai')
```

Comment:
Damn, nice catch. I didn't even notice that. I blame our product. Will fix in a couple minutes.